src/e2e/step-definitions/common.ts

```diff
@@ -12,7 +12,7 @@ Given("{string} navigates to {string} with path {string}", { timeout: 60 * 1000
 });
 
 Then("{string} sees the text: {string}", async function (this: CustomWorld, user, text) {
-    await this.page.waitForSelector(`text=${text}`);
+    await this.page.waitForSelector(`text=${text}`, {'timeout': 10000 });
```

Comment:
This 10 seconds right? Seems like a very long time.. I guess so but is this really needed? When I run ZAC locally everything runs pretty quickly.

Reply:
Agree 10 seconds is pretty long, but this is a change that will fix a flacky test in gh actions. sometimes creating a zaak takes over 5 seconds on test so the test will fail. This change is officially out of scope from this pr but i thought i might as well give test a little bit more room to perform these actions

Reply:
Ok! Wow, 5 seconds to create a zaak. That is indeed long. Would be good to check out what is going on. What I do know is that on GitHub those GitHub Action VMs are really quite small (as in resources). Nothing like your average MacBook so to speak.. ;-)